app/components/ui/NoSearchResultsDisplay.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import styles from "components/search/SearchResults/SearchResults.module.scss";
+import ClearSearchButton from "components/search/Refinements/ClearSearchButton";
+
+export const NoSearchResultsDisplay = ({ query }: { query: string }) => (
+  <div className={`${styles.noResultsMessage}`}>
+    <div className={styles.noResultsText}>
+      No results {query && `for ${` "${query}" `}`} found in your area.
+      <br /> Try a different location, filter, or search term.
+    </div>
+    {query && <ClearSearchButton />}
```

Comment:
**note:** While this component is used for the Browse Experience, theoretically there will never be a query in that case so the Clear Search button will not show.